.github/config/renovate.json5

```diff
@@ -17,6 +17,12 @@
     'config:recommended',
     'config:best-practices',
   ],
+  "packageRules": [
+    {
+      "matchPackageNames": ["github.com/docker/cli", "github.com/docker/docker"],
```

Comment:
this is the wrong location for this! please use the top level one in https://github.com/open-component-model/open-component-model/blob/main/renovate.json5

Also probably worth writing a glob expression for github.com/docker